plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/quickfix/createFromUsage/createCallable/CreateCallableFromCallActionFactory.kt

```diff
@@ -124,13 +133,15 @@ sealed class CreateCallableFromCallActionFactory<E : KtExpression>(
                         originalType
                     ).firstOrNull() ?: originalType
                 } else originalType
-                TypeInfo(finalType, Variance.IN_VARIANCE)
+                TypeInfo(finalType, Variance.IN_VARIANCE) to false
             }
             else -> throw AssertionError("Unexpected receiver: $receiver")
         }
     }
 
     protected fun getAbstractCallableInfo(mainCallable: CallableInfo, originalExpression: KtExpression): CallableInfo? {
+        if (mainCallable.isForCompanion) return null
```

Comment:
This `if` isn't covered with tests. If I comment it out and run tests then tests pass